server/filters/formatQualificationLevelFilter.ts

```diff
@@ -0,0 +1,16 @@
+export default function formatQualificationLevelFilter(value: string): string {
+  const levelValue = LevelValues[value as keyof typeof LevelValues]
+  return levelValue
+}
+
+enum LevelValues {
```

Comment:
I'm not 100% sure this is right, though I can see where it is in Gary's UI codebase so can understand why you've done it the way you have.
The problem is, the API returns us qualification levels that look like this:
```
    AchievedQualification: {
      /** @description This is the subject the inmate has chosen. */
      subject?: string
      /** @description This is the grade on the subject the inmate has chosen. */
      grade?: string
      /**
       * @description This is the level of  the subject the inmate has chosen.
       * @enum {string}
       */
      level?:
        | 'ENTRY_LEVEL_2'
        | 'ENTRY_LEVEL_3'
        | 'LEVEL_1'
        | 'LEVEL_2'
        | 'LEVEL_3'
        | 'LEVEL_4'
        | 'LEVEL_5'
        | 'LEVEL_6'
        | 'LEVEL_7'
        | 'LEVEL_8'
    }
```
IE. We get `ENTRY_LEVEL_2` and `ENTRY_LEVEL_3`; but no `ENTRY_LEVEL` ....

I'll talk to Gary and Sacintha about this apparent disconnect in their UI and API ... leave this one with me and I'll let you know what I find ...

Reply:
OK, I think what you've done is correct, but there is an API bug that will prevent this working in all cases!
https://mojdt.slack.com/archives/C056PJR1T2A/p1698333864249309

Dont worry about changing your version here, its correct. The CIAG dev team can fix the mismatch in their API (which is where the problem is)